Docs/Configuration.tex

```diff
@@ -1193,8 +1193,8 @@ \subsection{Introduction}\label{booterintro}
 \item \texttt{Above 4G Decoding} or similar enabled in firmware
   settings if present. Note, that on some motherboards (notably ASUS WS-X299-PRO) this
   option causes adverse effects, and must be disabled. While no other motherboards
-  with the same issue are known, consider this option to be first to check if one
-  encounters erratic boot failures.
+  with the same issue are known, this option should be checked first whenever erratic boot
```

Comment:
*whenever encountering boot failure*?

(This is sort of subject, and also failure is uncountable)

Reply:
Actually, it is.

Reply:
Fun... The Chinese-English/Japanese-English dictionary told me that it is uncountable when interpreted as lack of success, yet the Oxford English dictionary said that failure is countable. @_@